packages/itwinui-css/src/dialog/dialog.scss

```diff
@@ -171,6 +171,7 @@ $iui-dialogue-resizable-min-height: 7.75rem; // 7.75rem = 124px = title bar heig
   padding: 0 var(--iui-size-m);
   overflow-y: auto;
   overflow-y: overlay;
+  max-height: 70vh;
```

Comment:
i think instead of this, it might make more sense to set `max-height: 100%` on the dialog itself

Reply:
it seems like we already have a max-height set for the dialog. However the buttons would still be unreachable. I'm not sure why but I think probably because they are flex relative to the dialog.
![image](https://user-images.githubusercontent.com/22462084/217662148-6d394008-5d3d-4b3f-8c33-0e03a2a0a4d6.png)

Which is why I thought it'd be better to have max-height equal 100%/inherit minus [buttons height]? But setting smaller dialog max-height would cause the content to flow out like this 
![image](https://user-images.githubusercontent.com/22462084/217663366-c3c9f97f-1a1d-4848-8c2c-2caad697dad8.png)

Which circle back to why I was messing with the content max-height instead of the dialog 🤔

Reply:
possibly related: i noticed the same issue happens with width too.
![](https://user-images.githubusercontent.com/9084735/217664298-32fa9c37-c1ed-4af7-b276-474327d7e2f3.png)


whereas full page dialog looks fine: 
![](https://user-images.githubusercontent.com/9084735/217664430-66abfcc6-e241-48c4-b8ee-69d77a3d373f.png)

so we might want to use whatever approach full page dialog is using.

Reply:
The width issue is fixed by changing line 53:

```diff
-  min-width: $iui-dialog-width;
+  min-width: Min(100%, $iui-dialog-width);
```

For the height, I just realized we didn't add `display: flex` on the regular dialog to keep backwards compatibility with an older version where `Dialog.Content` was optional.

Reply:
Try something like this on the dialog-content:
```css
max-height: calc(100% - 7.5rem);
```

you'll have to calculate the correct value manually.

Reply:
dialog-content max-height as a percentage do not seem to have any effect (also tried calc too)
![image](https://user-images.githubusercontent.com/22462084/217668243-e0c9449b-e190-4bfc-a2f9-425c31f6fe0f.png)
![image](https://user-images.githubusercontent.com/22462084/217668096-17f433bf-8cff-4ae1-92a8-e2f686c21109.png)

but vh is good
![image](https://user-images.githubusercontent.com/22462084/217668372-bff78622-3aa6-49f5-9286-cdbacedcbc50.png)

Reply:
I think we could just do 100vh - [buttons size] ?